<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>POI_delete_ui</class>
 <widget class="QDialog" name="POI_delete_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection de type de marque</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>210</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lnk_all">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;titi&quot;&gt;Tous&lt;/a&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lnk_none">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;toto&quot;&gt;Aucun&lt;/a&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="chk_POI">
          <property name="text">
           <string>POI</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_WP">
          <property name="text">
           <string>Route</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_Balise">
          <property name="text">
           <string>Balise</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type de marque Ã  supprimer</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>chk_POI</tabstop>
  <tabstop>chk_WP</tabstop>
  <tabstop>chk_Balise</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lnk_all</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>POI_delete_ui</receiver>
   <slot>do_chkAll(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lnk_none</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>POI_delete_ui</receiver>
   <slot>do_chkNone(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>POI_delete_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>POI_delete_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>do_chkAll(QString)</slot>
  <slot>do_chkNone(QString)</slot>
 </slots>
</ui>
